### PR TITLE
Feat (config): Add --allowEmpty option

### PR DESCRIPTION
**Pull request by emir7**
A potential solution for [#4197](https://github.com/stryker-mutator/stryker-js/issues/4197)

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Keep in mind, that each option used on the command line will **completely replac
 ## Usage of Globbing Expressions on options
```
Comment (nicojs) [inline docs/configuration.md:13]: I see you've formatted this file with something. Please undo that, as this will pollute our history.

**Review by nicojs — CHANGES_REQUESTED** [review]
This looks great. Just a small remark.

File: docs/configuration.md (hunk 1 above)
Comment (emir7) [reply docs/configuration.md]: not sure why the file was exactly formatted, but it should be fine now

Comment (nicojs) [reply docs/configuration.md]: Thanks! It's not your fault though, it is normal to use a formatted in your editor. We should use formatter to verify it in the pipeline...

**Review by nicojs — APPROVED** [review]

**Comment by emir7** [comment]
Seems like the tests are failing for some reason on Windows 😅 the error is a little bit odd and is related to the `vitest-runner`. It prints the following: 
`Error: EBUSY: resource busy or locked, rmdir 'D:\a\stryker-js\stryker-js\packages\vitest-runner\testResources\tmp\workDir-7081502'`
not 100% sure, but it looks like it might something to do with the fact that processes aren't fully closed in pipelines during the test run

**Comment by nicojs** [comment]
Yeah, probably. I think it's a bug in Vitest itself (we're calling and awaiting `.close()`) We've just merged the Vitest runner and I'm seeing these errors for the first time... I'll restart the builds

**Pull request merged**